src/components/common-components/modal/modal.component.tsx

```diff
@@ -11,71 +11,79 @@ import { Box, Divider, Grid, IconButton } from "@material-ui/core";
 import { MdClose } from "react-icons/md";
 import ScssVars from "../../../assets/styles/styles/custom/_variables.module.scss";
 
-const useStyles = makeStyles((theme: Theme) =>
-  createStyles({
-    modal: {
-      display: "flex",
-      alignItems: "center",
-      justifyContent: "center",
-      position: "relative",
-    },
-    paper: {
-      backgroundColor: "#FFFFFF",
-      border: "0",
-      boxShadow: "-3px 4px 20px 4px rgba(0, 0, 0, 0.15)",
-      padding: theme.spacing(2, 1, 3),
-      maxWidth: "600px",
-      maxHeight: "350px",
-      minWidth: "424px",
-      minHeight: "198px",
-    },
-    titleMargin: {
-      paddingTop: 15,
-      paddingLeft: 25,
-      paddingBottom: 15,
-    },
+const useStyles = (
+  height: number,
+  width: number
+): Record<
+  "modal" | "paper" | "titleMargin" | "modalBody" | "footer" | "exitButton" | "title",
+  string
+> =>
+  makeStyles((theme: Theme) =>
+    createStyles({
+      modal: {
+        display: "flex",
+        alignItems: "center",
+        justifyContent: "center",
+        position: "relative",
+      },
+      paper: {
+        backgroundColor: "#FFFFFF",
+        border: "0",
+        boxShadow: "-3px 4px 20px 4px rgba(0, 0, 0, 0.15)",
+        padding: theme.spacing(2, 1, 3),
+        maxWidth: `${height}px`,
+        maxHeight: `${width}px`,
+        minWidth: "424px",
+        minHeight: "198px",
```

Comment:
Btw, to można by było to zrobić tak, żeby można było przekazywać `minWidth` i `minHeigth`. 
Ewentualnie, można też zrobić tak, że to będzie liczone automatycznie na podstawie maxWith/maxHeight.